core/themes/seven/css/style.css

```
@@ -808,6 +808,28 @@ details .details-child-wrapper {
   margin: 10px 0;
 }
 
+/**
+ * Description version of details element.
+ */
+details.description {
+  padding: 6px 15px;
+  border-radius: 0 0 4px 4px;
+  border: 2px solid #EAEAEA;
```

We always use lowercase in hex colors in CSS:

```suggestion
  border: 2px solid #eaeaea;
```

---

Ah; that's residue from when we used the details element in this PR. We're no longer using it in the latest PR, so I will remove that styling. @jenlampton (who introduced it) might want to re-introduce it as a general styling issue for details element as a new issue.